Web_Development/FrontEnd/SEO_Basics/README.md

```diff
@@ -0,0 +1,23 @@
+# SEO/Search Engine Optimization
+----
+## What is it?
+ ### It is the process of improving your website's visibility when people search for products or services related to your business in any Search Engine. The better visibility your pages have in search results, the more likely you are to gain the attention of customers to your business.
+ <img src="img/SEO_img.png" width="400"><br>
+ ----
+ ## How does it work?
+ ### In order to understand this let's take the example of the Google Search Engine.
+ - ### Google stores all its web pages in an ***index***. Every ***index*** entry has a Description and URL assiciated with it. *To index* is when Google fetches a page, reads it, and adds it to the index: *Google indexed several pages on my site today*.
+ - ### Google *crawls* the web, looking for new pages, then indexes them if necessary with the help of its *crawler*- the *GoogleBot*. GoogleBot is an automated software that acts as a librarian to help us find what we are searching for.<br>
+ ----
+### The techniques and factors which work in co-relation with each other to improve the overall SEO of your website are known as the SEO Success Factors. The Periodic Table of SEO Factors organizes the factors into six main categories and weights each based on its overall importance to SEO. In this section our main focus would be on the factors related to HTML Codes.<br>HTML is the coding standard used to make most web pages. Search engines can pick up specific signals from various HTML elements. So, the designers must use these HTML elements effectively to gain maximum advantage.<br><br>Here are some of the HTML elements to consider for SEO.

```

Comment:
`###` makes all content like a heading. Could use normal text at some areas like these.

Reply:
I have made the changes 
is it okay now?